Ghidra/RuntimeScripts/Linux/support/launch.sh

```diff
@@ -123,6 +123,11 @@ if [ "${MAXMEM}" != "" ]; then
 	VMARG_LIST+=" -Xmx${MAXMEM}"
 fi
 
+# if using jdk 14 or later enable helpful NullPointerException
+if [ $(java -version 2>&1 | sed -E -n 's/.* version "([^.-]*).*"/\1/p' | cut -d' ' -f1) -ge 14 ]; then
```

Comment:
You need to be checking the version of $JAVA_CMD.  That will ultimately be the java that Ghidra ends up using.

Reply:
I have modified the changes to instead append both `XX:+IgnoreUnrecognizedVMOptions` and `-XX:+ShowCodeDetailsInExceptionMessages` to the vmargs. Should the vmargs be checked first for an unrecognized option from the user before appending these?